powerquery-docs/connectors/smartsheet.md

```diff
@@ -15,18 +15,54 @@ ms.author: dougklo
 | ---- | ----------- |
 | Release State | General Availability |
```

Comment:
This connector is not in general availability. Please change this to be in Beta and match the current status of your connector